src/components/yemen/education/resources.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import IRC from './IRC.png'
+import Iran from './Iran.png'
+import Crisis from './Crisis.png'
```

Comment:
Please look at how I usually import images: https://github.com/amanmibra/actfor/blob/master/src/components/yemen/donate/index.jsx#L11

and follow that. Also please, have all of your images in the `images` folder.